config/webhook/templates_webhook.yaml

```diff
@@ -0,0 +1,91 @@
+# Copyright 2021 VMware
```

Comment:
it's not new in this PR, but is there a reason we are hand rolling the webhook config rather than using controller-gen to generate it from the go source?

Reply:
Done in #747